tests/components/unifi/test_switch.py

```diff
@@ -340,14 +344,17 @@
     assert len(controller.mock_requests) == 3
     assert len(hass.states.async_all()) == 2
 
-    controller.mock_client_responses.append([CLIENT_1, CLIENT_2])
-    controller.mock_device_responses.append([DEVICE_1])
+    controller.api.websocket._data = {
+        "meta": {"message": "sta:sync"},
+        "data": [CLIENT_2],
+    }
+    controller.api.session_handler("data")
 
     # Calling a service will trigger the updates to run
     await hass.services.async_call(
         "switch", "turn_off", {"entity_id": "switch.poe_client_1"}, blocking=True
     )
-    assert len(controller.mock_requests) == 6
+    assert len(controller.mock_requests) == 4
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4925854)